osuapi/enums.py

```diff
@@ -147,12 +149,16 @@ class BeatmapGenre(Enum):
     novelty = 7
     hip_hop = 9
     electronic = 10
+    metal = 11
+    classical = 12
+    folk = 13
+    jazz = 14
 
 
 class BeatmapLanguage(Enum):
     """Enum represeting the language of a beatmap."""
     any = 0
-    other = 1
+    unspecified = 1
```

Comment:
So was the old api inaccurate? have you tried fetching sample beatmap sets and checking that the api number matches? If this is correct, cool, it's just strange it contradicts the wiki

Reply:
Yeah, no problem.
[This](https://osu.ppy.sh/api/get_beatmaps?b=2011121) [(beatmap)](https://osu.ppy.sh/beatmapsets/960601#osu/2011121) returns `"language_id":"1"`, unspecified.
[This](https://osu.ppy.sh/api/get_beatmaps?b=1991102) [(beatmap)](https://osu.ppy.sh/beatmapsets/945922#osu/1991102) returns `"language_id":"14"`, other.
[This](https://osu.ppy.sh/api/get_beatmaps?b=1906448) [(beatmap)](https://osu.ppy.sh/beatmapsets/912629#osu/1906448) returns `"language_id":"13"`, polish.

My guess is that when they introduced the 'unspecified' category they moved 'other' to another number... maybe